Makefile

```
@@ -8,9 +8,16 @@ REGRESS = set_user
 
 LDFLAGS_SL += $(filter -lm, $(LIBS))
 
+ifdef WINDOWS_DISUSE_PGXS
```

Since the non-pgxs build method still seems to be a thing some people are using, and that's all this is really providing an option for, I think it's probably good to just have this be the mode option for now. If things need to be more specialized for Windows, we can make those additions if/when that happens.
```suggestion
ifdef NO_PGXS
```

---

Sounds great to me!